### ISSUE TITLE
Links in profile page fail to meet WCAG 2.0 AA compliance for SC 1.4.3

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/vets-design-system-documentation#365]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

![image](https://user-images.githubusercontent.com/14154792/103230572-ca23b780-4903-11eb-8de2-5bb3cf4700ea.png)
The links in the profile page [do not meet accessibility requirements for contrast](https://www.w3.org/TR/WCAG20-TECHS/G183.html).

## What I expected to happen

Styling these as buttons will address the issue.

## Reproducing
- Formation version: --
- Device: any
- Browser: any

Steps to reproduce:

1. Open up profile page in staging
